nodejs/week3/homework-template/src/warmup/index.js

```diff
@@ -0,0 +1,101 @@
+const express = require("express");
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.set('json spaces', 2); // prettify json output
+
+app.get("/calculator/:method", (req, res) => {
+  if (req.params.method === "add") {
+    let result = addNumbers(req);
```

Comment:
Really nice! Now i'm getting picky. But its a little weird that a number that is called addNumbers take a req. I would expect that function to take an array of numbers. 

Like what if i wanted to use the function in another part of the application but dont have a request. 

So maybe you could do `addNumbers(Object.values(req))` Now everything is how we expect it :)